features/promotion/applying_catalog_promotions/reapplying_multiple_catalog_promotions_after_editing_their_channels.feature

```diff
@@ -0,0 +1,38 @@
+@applying_catalog_promotions
+Feature: Reapplying multiple catalog promotions after editing their channels
+    In order to have proper discounts in different channels
+    As a Store Owner
+    I want to have discounts reapplied in product catalog once the channels of catalog promotion changes
+
+    Background:
+        Given the store operates on a channel named "Web-US" with hostname "web-us"
+        And the store operates on another channel named "Web-GB" with hostname "web-gb"
+        And the store has a "T-Shirt" configurable product
+        And this product has "PHP T-Shirt" variant priced at "$100.00" in "Web-US" channel
+        And "PHP T-Shirt" variant priced at "$30.00" in "Web-GB" channel
+        And this product is available in "Web-US" channel and "Web-GB" channel
+        And there is a catalog promotion "Winter sale" available in "Web-US" channel that reduces price by "30%" and applies on "PHP T-shirt" variant
+        And there is a another catalog promotion "Christmas sale" available in "Web-US" channel and "Web-GB" channel that reduces price by "50%" and applies on "PHP T-shirt" variant
+        And I am logged in as an administrator
+
+    @api @todo
+    Scenario: Removing only modified catalog promotion after removing its channel
```

Comment:
This scenario looks the same as the one from `features/promotion/applying_catalog_promotions/reapplying_catalog_promotions_after_editing_its_channels.feature:18`. 
I would be for modifying it to have variant with 2 catalog promotions applied and remove only one of them by changing channel or just remove a channel from `Web-GB` channel 🤔

Reply:
I'm resolving as it was an invalid comment 🎉

Reply:
they are similar but not the same, the main difference is in line 15 (where I've configured second CP)